snapcraft/internal/meta/command.py

```diff
@@ -195,10 +219,25 @@ def prime_command(
         if self.requires_wrapper:
             if not can_use_wrapper:
                 raise errors.InvalidAppCommandFormatError(self.command, self._app_name)
+            if self.original_command.startswith("$SNAP"):
+                logger.warning(_FMT_SNAP_STRIP.format(self.original_command))
+            if self.command.startswith("/") and not self.original_command.startswith(
+                "/"
+            ):
+                logger.warning(
```

Comment:
I think this error will be odd if this happens due to the interpreter being picked up?

Reply:
Good catch

Reply:
Good catch, this might be tricky

Reply:
This will not happen, we raise `errors.ShebangInRoot` for that scenario, which avoids the wrapper